src/gui/MItem_menus.cpp

```diff
@@ -402,17 +402,32 @@ MI_LOAD_SETTINGS::MI_LOAD_SETTINGS()
     : WI_LABEL_t(_(label), nullptr, is_enabled_t::yes, is_hidden_t::no) {
 }
 
+static void add_load_response(const char *name, bool flag, char *msg, int msglen) {
+    char tmp[strlen(name) + 5];
+    snprintf(tmp, sizeof(tmp), "%s: %s\n", name, flag ? "+" : "-");
```

Comment:
"+" and "-" are not very telling. If you want to make it short, make it for example "Ok" and "Fail". We're not that limited in memory.

Also, the "Ok" and "Fail" need to be translated

Reply:
Changed that.